src/components/DisplayButton.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+
+class DisplayButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  showCorrectValue() {
+    const { calculator } = this.props;
+    if (calculator.next) {
+      return calculator.next;
+    }
+    if (calculator.total) {
+      return calculator.total;
+    }
+    return 0;
+  }
+
+  render() {
+    return (
+      <button type="button" className="grey">
+        {this.showCorrectValue()}
+      </button>
```

Comment:
- [ ] [OPTIONAL] Can please fix the alignment of the result? for now, the result is centered whereas it could be aligned to the right 👍